CAPABILITIES.md

```diff
@@ -6,24 +6,20 @@ those capabilities for the Ruby SDK:
 ## Developer Wallets
 
 | Concept       | Base-Sepolia | Base-Mainnet | Ethereum-Holesky | Ethereum-Mainnet |
-| ------------- | :----------: | :----------: | :--------------: | :--------------: |
-| Addresses     |      ✅      |      ✅      |        ✅        |        ✅        |
-| Send          |      ✅      |      ✅      |        ✅        |        ❌        |
-| Trade         |      ❌      |      ✅      |        ❌        |        ❌        |
-| Faucet        |      ✅      |      ❌      |        ✅        |        ❌        |
-| Server-Signer |      ✅      |      ✅      |        ❌        |        ❌        |
-| Stake [^1]    |      ❌      |      ❌      |        ✅        |        ✅        |
-
-[^1]: Currently only available for Shared ETH Staking.
+|---------------|:------------:|:------------:|:----------------:|:----------------:|
+| Addresses     |      ✅       |      ✅       |        ✅         |        ✅         |
+| Send          |      ✅       |      ✅       |        ✅         |        ❌         |
+| Trade         |      ❌       |      ✅       |        ❌         |        ❌         |
+| Faucet        |      ✅       |      ❌       |        ✅         |        ❌         |
+| Server-Signer |      ✅       |      ✅       |        ❌         |        ❌         |
+| Stake         |      ❌       |      ❌       |        ✅         |        ✅         |
```

Comment:
@jazz-cb we should update this, wasn't aware we had this totally additional file as well 😅 

Probably should not have this live in two different places